api/graphql/schema.graphql

```diff
@@ -532,6 +544,24 @@ type Role {
   displayName: LocalizedString @rename(attribute: "display_name")
   description: LocalizedString
   isTeamBased: Boolean @rename(attribute: "is_team_based")
+  # teams and roles: built-in
+  users: [UserPublicProfile!] @morphToMany
+  permissions: [Permission!] @belongsToMany
```

Comment:
I don't think we have any use for querying permissions yet. I think this could be deleted.

Reply:
Yeah, let's not add things we don't need yet.  Removed in [remove extra relationships](https://github.com/GCTC-NTGC/gc-digital-talent/pull/5749/commits/69500c02baa6a090cdde1213656bb8672c78251c).